features/data/configs/upgrade_from_195_with_missing_journal.json

```diff
@@ -0,0 +1,11 @@
+{
+"default_hour": 9,
+"timeformat": "%Y-%m-%d %H:%M",
+"linewrap": 80,
+"encrypt": false,
```

Comment:
Can we add the same tests but for an encrypted journal? Just be sure it works.

Reply:
I've added a test for this which is failing, although the actual code works.  I'm not certain, but it could be that the output it being piped to stderr which doesn't have an equivalent "the output should contain".  Can you take a look?  I'll add a step in core.py if so.